BigBaseV2/src/backend/looped/self/no_ragdoll.cpp

```diff
@@ -5,13 +5,6 @@ namespace big
 {
 	void looped::self_no_ragdoll()
 	{
-		Ped player = PLAYER::PLAYER_PED_ID();
-
-		if (g->self.no_ragdoll) {
-			PED::SET_PED_CAN_RAGDOLL(player, !g->self.no_ragdoll);
-			PED::SET_PED_CAN_RAGDOLL_FROM_PLAYER_IMPACT(player, !g->self.no_ragdoll);
-			PED::SET_PED_RAGDOLL_ON_COLLISION(player, !g->self.no_ragdoll);
-		}
-			
```

Comment:
You sure you want to always run this native, you're even executing the native when the feature is disabled, possibly causing side effects in missions. Have you properly tested this?